### ISSUE TITLE
'tmt run -avvv provision -h minute -i f36 test --name .' generates traceback

### ISSUE THREAD

**Issue by zdohnal** [teemtee/tmt#1699]
Hi,

I wanted to check whether my test works with the command from subject and got the following traceback:

```
$ tmt run -avvv provision -h minute -i f36 test --name .
/var/tmp/tmt/run-007
warn: /plans/all: - Additional properties are not allowed ('contact' was unexpected)
Found 1 plan.

/plans/all
summary: Run all tests
    discover
        how: fmf
        order: 50
        directory: /home/zdohnal/fedtests_repo/vim
        hash: 653d1a6
        summary: 1 test selected
            /Sanity/upstream-unittests
    provision
        how: minute
        order: 50
        image: f36
        flavor: m1.small
        auto-select-network: False
        skip-prereserve: False
    finish
        summary: 0 tasks completed
Traceback (most recent call last):
  File "/usr/bin/tmt", line 20, in <module>
    tmt.cli.main()
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1691, in invoke
    return _process_result(rv)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1628, in _process_result
    value = ctx.invoke(self._result_callback, value, **ctx.params)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/lib/python3.10/site-packages/tmt/cli.py", line 330, in finito
    click_context.obj.run.go()
  File "/usr/lib/python3.10/site-packages/tmt/base.py", line 2572, in go
    plan.go()
  File "/usr/lib/python3.10/site-packages/tmt/base.py", line 1526, in go
    step.go()
  File "/usr/lib/python3.10/site-packages/tmt/steps/provision/__init__.py", line 1158, in go
    phase.go()
  File "/usr/lib/python3.10/site-packages/tmt/steps/provision/minute.py", line 292, in go
    self._guest = GuestMinute(data, name=self.name, parent=self.step)
TypeError: Guest.__init__() takes 1 positional argument but 2 positional arguments (and 2 keyword-only arguments) were given
```

would you mind looking into it?

My version is ```tmt-1.20.dev-1.20221122165818832033.main.8.gdc1bc1a.fc36.noarch```, a colleague has ```tmt-1.19.0-1.fc36.noarch``` which works.

**Comment by psklenar** [comment]
fyi I have the same traceback with f37: tmt-1.20.dev-1.20221121193449725496.main.7.gbe1093a.fc37.noarch

**Comment by lukaszachy** [comment]
Fixed with tmt-redhat-provision-minute-1.5

**Comment by lukaszachy** [comment]
Sorry, slipped and I didn't pay enough attention to investigate failing (internal) nightly test
